File: app/modules/computer/src/main/java/com.multiapk.modules.computer/ComputerFragment.java
package com.multiapk.modules.computer;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;
import android.widget.Toast;

import com.multiapk.library.base.MFragment;


public class ComputerFragment extends MFragment {

    @SuppressLint("InflateParams")
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View contentView = inflater.inflate(R.layout.fragment_computer, null);
        TextView textHospital = (TextView) contentView.findViewById(R.id.textComputer);
        textHospital.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(mFragmentActivity, "module-computer", Toast.LENGTH_SHORT).show();
            }
        });
        return contentView;
    }
}

File: android/arsenal/modules/module-computer/src/main/java/com.multiapk.modules.computer/ComputerFragment.kt
package com.multiapk.modules.computer

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import org.jetbrains.anko.toast
import org.smartrobot.base.DefaultBaseFragment

class ComputerFragment : DefaultBaseFragment() {
    @SuppressLint("InflateParams")
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val contentView = inflater!!.inflate(R.layout.computer_fragment_computer, null)
        context?.toast("cici is nice")
        /*RxView.clicks(contentView).subscribe {
            context?.toast("电脑模块" + Thread.currentThread().name)
        }*/
        return contentView
    }
}
